staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/authentication.go

```diff
@@ -31,17 +31,18 @@ import (
 // rest.Config generated by the resolver.
 type AuthenticationInfoResolverWrapper func(AuthenticationInfoResolver) AuthenticationInfoResolver
 
-// AuthenticationInfoResolver builds rest.Config base on the server name.
+// AuthenticationInfoResolver builds rest.Config base on the server name and
+// the isService flag indicating whether the webhook is deployed as a service.
```

Comment:
The thing we want this name to highlight is whether the server is routable directly from apiserver's network environment. `isService` isn't the right name. Maybe `indirectRouting`? Note that the ssh tunnels are going away in less than a year, so please leave a TODO to remove this when they go away.

Reply:
Done. Changed the variable name to `directRouting` and added TODO.